lib/winston/transports/console.js

```diff
@@ -21,15 +21,52 @@ var Console = exports.Console = function (options) {
   Transport.call(this, options);
   options = options || {};
 
-  this.json        = options.json        || false;
-  this.colorize    = options.colorize    || false;
-  this.prettyPrint = options.prettyPrint || false;
-  this.timestamp   = typeof options.timestamp !== 'undefined' ? options.timestamp : false;
-  this.showLevel   = options.showLevel === undefined ? true : options.showLevel;
-  this.label       = options.label       || null;
-  this.logstash    = options.logstash    || false;
-  this.debugStdout = options.debugStdout || false;
-  this.depth       = options.depth       || null;
+  this.json         = options.json        || false;
+  this.colorize     = options.colorize    || false;
+  this.prettyPrint  = options.prettyPrint || false;
+  this.timestamp    = typeof options.timestamp !== 'undefined' ? options.timestamp : false;
+  this.showLevel    = options.showLevel === undefined ? true : options.showLevel;
+  this.label        = options.label       || null;
+  this.logstash     = options.logstash    || false;
+  this.stderrLevels = (function (levels, debugStdout) {
```

Comment:
An immediately invoked function expression is a rather cumbersome way to do this. Could you clean it up a bit?